### ISSUE TITLE
navigation to sync - too many button presses

### ISSUE THREAD

**Issue by stefangweichinger** [memen45/SubMusic#10]
I create a new issue for this because it's annoying in daily usage.

As mentioned below I can't tell if it is specific to the watch, caused by the firmware or your app. I'd just like to give feedback, maybe I can learn some different access method or you know a way to improve this:

```
I always have to switch between "My Music" and "SubMusic" as music provider to be able to access the sync-menu again.
Otherwise it goes straight into "Playlists" from where I don't know how to get to syncing or testing.

This means many buttons to press each time ... Maybe it is related to the Firmware of the Fenix 6, I don't know.
```

_Originally posted by @stefangweichinger in https://github.com/memen45/SubMusic/issues/6#issuecomment-711711868_

**Comment by memen45** [comment]
Thanks for your feedback. I recognise the problem. It seems dependent on the device however, since I have a separate sync button in the menu (vivoactive 4s). There is a library icon (it is the app icon) and a sync icon (music note with arrows around it). The latter one brings you to the sync menu.

The problem I have is that a sync initiated from anywhere except the official sync button seems to fake sync (everything completes, but files are not downloaded). Additionally, syncs are performed in the background by the watch as well from time to time.

I will try and look into this for the next release!

**Comment by stefangweichinger** [comment]
Yes, this looks different on the F6.

**Comment by memen45** [comment]
This is included in version 0.0.22-chi, released just now in the Garmin app store.

Thanks a lot for your sharing your idea!
